### PR TITLE
release7.15.0

### PR DESCRIPTION
**Pull request by sandraros**

### CODE REVIEW

File: src/zcl_excel.clas.abap [hunk 1]
```diff
@@ -156,7 +156,7 @@ CLASS zcl_excel DEFINITION
     DATA worksheets TYPE REF TO zcl_excel_worksheets .
   PRIVATE SECTION.
 
-    CONSTANTS version TYPE c LENGTH 10 VALUE '7.14.0'.      "#EC NOTEXT
+    CONSTANTS version TYPE c LENGTH 10 VALUE '7.15.0'.      "#EC NOTEXT
```
Comment (AndreaBorgia-Abo) [inline src/zcl_excel.clas.abap]: if we move this definition to the public section, a move that would make sense in its own right for such a field, then...

File: src/zcl_excel_apack.clas.abap [hunk 2]
```diff
@@ -23,7 +23,7 @@ CLASS zcl_excel_apack IMPLEMENTATION.
 
     descriptor-group_id        = 'github.com/abap2xlsx/abap2xlsx'.
     descriptor-artifact_id     = 'abap2xlsx'.
-    descriptor-version         = '7.14.0'.
+    descriptor-version         = '7.15.0'.
```
Comment (AndreaBorgia-Abo) [inline src/zcl_excel_apack.clas.abap]:
```suggestion
    descriptor-version         = zcl_excel=>version.
```

Comment (AndreaBorgia-Abo) [reply src/zcl_excel_apack.clas.abap]: ...here we could reference and always be ok.

**Review by AndreaBorgia-Abo — APPROVED** [review]
LGTM.
One quick, process-related confirmation: are we using the GH generated changelog as basis or is the changelog creation entirely manual?

**Comment by sandraros** [comment]
Currently, both systems exist and are maintained separately. I update `changelog.txt` manually from the list of latest commits. One is for seeing the changes from within ZABAPGIT (APACK), one is for seeing it directly on GitHub via Release management.

**Review by AndreaBorgia-Abo — APPROVED** [review]

**Comment by AndreaBorgia-Abo** [comment]
To be clear, I intentionally did not merge right away because I saw that multiple reviewers were involved and I'm not sure if you wanted feedback from all of us. Feel free to merge if one approval is enough.

**Comment by sandraros** [comment]
Thanks. We didn't agree on a policy for the releases, so I think we can merge. I just propose that we create a release from time to time, every 1 or 2 months for instance, to contain enough changes, but not too much.

**Pull request merged**